core/src/main/scala/wartremover/contrib/warts/SymbolicName.scala

```diff
@@ -0,0 +1,27 @@
+package org.wartremover
+package contrib.warts
+
+object SymbolicName extends WartTraverser {
+
+  private val validChars = "[a-zA-Z_]+".r
+
+  def apply(u: WartUniverse): u.Traverser = {
+    import u.universe._
+
+    def isSymbolic(name: String) = validChars.replaceAllIn(name, "").length > 2
```

Comment:
Sorry for the silly autocorrect. I mean in scala community `===` is the chosen symbol.

Anyways I'm not against the wart, as long as you mention the criteria in readme